cviceni2/toth/prvni_skript.py

```diff
@@ -0,0 +1,55 @@
+import geopandas as gpd # Import the Geopandas package
+import matplotlib.pyplot as plt # Import the Matplotlib package
+import matplotlib.patches as mpatches # Import the Matplotlib package (patches module)
+
+# Set the population threshold values
+thresholds = [100000000, 300000000, 500000000]
+
+# Loop through the threshold values
+for threshold in thresholds:
+    # Load the world map
+    world = gpd.read_file(gpd.datasets.get_path('naturalearth_lowres'))
+
+    # Assign colors to each continent for differentiation
+    # Key: Continent name, Value: Assigned color
+    continent_colors = {
+        'Africa': '#ffcc99',
+        'Europe': '#99ccff',
+        'Asia': '#ff9999',
+        'North America': '#99ff99',
+        'South America': '#ffff99',
+        'Oceania': '#cc99ff',
+        'Antarctica': '#aaa'
+    }
+
+    # Plotting
+    fig, ax = plt.subplots(1, 1, figsize=(15, 10))
+
+    # Plot each continent with its assigned color, one by one
+    for key in continent_colors:
+        world[world['continent'] == key].plot(ax=ax, color=continent_colors[key])
+
```

Comment:
Super! Druhá možnost je ale ještě použít "for key, value in continent_colors.items():"